dev/NavigationView/TestUI/NavigationView_TestUI.projitems

```diff
@@ -54,10 +58,6 @@
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
```

Comment:
Why did we have RS3 and RS4 specific pages ?

Reply:
See my answer above